src/applications/disability-benefits/new-686/config/chapters/report-child-stopped-attending-school/child-information/childInformation.js

```diff
@@ -0,0 +1,37 @@
+import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+
+import { genericSchemas } from '../../../generic-schema';
+import { validateName } from '../../../utilities';
+
+const { fullName, date } = genericSchemas;
+
+export const schema = {
+  type: 'object',
+  required: ['childNoLongerAtSchoolName', 'dateChildLeftSchool'],
```

Comment:
We can't set fields to be required in the schema, this will cause the form to silently not be able to submit like the rainbows team form did. Required fields needs to be set in the uiSchema only